util/Makefile

```diff
@@ -4,17 +4,24 @@ TARGET = zenfs
 
 CC ?= gcc
 CXX ?= g++
+OBJCOPY ?= objcopy
 
 CXXFLAGS = $(shell pkg-config --cflags rocksdb)
 LIBS = $(shell pkg-config --static --libs rocksdb)
 
 CXXFLAGS +=  $(EXTRA_CXXFLAGS)
 LDFLAGS +=  $(EXTRA_LDFLAGS)
 
-all: $(TARGET)
+all: $(TARGET) generate_dbg
```

Comment:
```suggestion
all: $(TARGET) $(TARGET).dbg
```

Reply:
Removed the generate_dbg.

> I think sign-off is formatted wrong. You need a space between name and `<` to make the CI happy

Ok,  formatted correctly.